src/library/simplified/extension/state-province-code.js

```diff
@@ -56,4 +56,8 @@ else if (utag.isMCO()) {
     else if (b["entity.checkout.cars.0.dropOffLocation.address.province"] != undefined) {
         b["stateProvinceCode"] = b["entity.checkout.cars.0.dropOffLocation.address.province"];
     }
+}
+else if (utag.isDestinationLandingPage() && b.entity.tripDetails.hotelOffer.hotelDescription.gaiaRegion.localizedRegionName != undefined)
+{
+    b['stateProvinceCode'] = b['entity.tripDetails.hotelOffer.hotelDescription.gaiaRegion.localizedRegionName'];
```

Comment:
Seems it is hack, I had to look into DX app to see you have piggy backed on this uis field
            hotelOffer.getHotelDescription().getGaiaRegion().setLocalizedRegionName(flexContext.getSearchContext().getLocation().getParent().getName());